### PR TITLE
Deploy Elasticsearch cluster with new lists deployed

### PR DESCRIPTION
**Pull request by jonnyshaw89**
I have changed this from a Terraform module to a standalone AWS deployment so that it can stand independently of Survey-Runner.

The new lists are populated with both the `en` and `cy` values.

The related `eq-lookup-api` will be required to test the lookups.

### CODE REVIEW

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Errored when trying to plan.
`Error: module.eq-lookup-ecs.data.aws_ami.amazon_ecs_ami: "owners": required field is not set`

**Review by MebinAbraham — COMMENTED** [review]
Worth running `terraform fmt` once complete.

**Comment by jonnyshaw89** [comment]
@MebinAbraham What terraform versions are you running?

**Comment by MebinAbraham** [comment]
@jonnyshaw89 `Terraform v0.10.8` As specified in `.terraform-version` file.

**Comment by jonnyshaw89** [comment]
@MebinAbraham what version of the aws provider?

**Comment by MebinAbraham** [comment]
@jonnyshaw89 Working as of latest push.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Can't successfully apply.
```
null_resource.load-data (local-exec): + echo 'Inserting Bulk List'
null_resource.load-data (local-exec): Inserting Bulk List
null_resource.load-data (local-exec): + curl -XPOST 'https://search-mebin-suggest-rupsujkquix7ygfj5f3rzmyilq.eu-west-1.es.amazonaws.com/*/*/_bulk' -H 'Content-Type: application/json' --data-binary '@built-code-lists/*.json'
null_resource.load-data (local-exec): Warning: Couldn't read data from file "built-code-lists/*.json", this makes an
null_resource.load-data (local-exec): Warning: empty POST.
null_resource.load-data (local-exec):   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
null_resource.load-data (local-exec):                                  Dload  Upload   Total   Spent    Left  Speed
null_resource.load-data (local-exec):   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
null_resource.load-data (local-exec): 100   163  100   163    0     0    227      0 --:--:-- --:--:-- --:--:--   227
null_resource.load-data (local-exec): 100   163  100   163    0     0    227      0 --:--:-- --:--:-- --:--:--   227
null_resource.load-data (local-exec): {"error":{"root_cause":[{"type":"parse_exception","reason":"request body is required"}],"type":"parse_exception","reason":"request body is required"},"status":400}
null_resource.load-data: Creation complete after 1s (ID: 4697778277594374335)

Error: Error applying plan:

1 error(s) occurred:

* module.eq-lookup.data.aws_lb.eq: data.aws_lb.eq: Search returned 3 results, please revise so only one is returned
```

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Getting 500s when making an request:
```
  File "/Users/abrahm/Documents/projects/eq-lookup-api/app/__init__.py", line 20, in search
    results = get_results(index, language, query)
  File "/Users/abrahm/Documents/projects/eq-lookup-api/app/request_lookup.py", line 44, in get_results
    raise Exception("ERROR IN LOOKUP API.")
Exception: ERROR IN LOOKUP API.
```
Request:
```
127.0.0.1 - - [30/Apr/2019 08:29:17] "GET /country/?q=India HTTP/1.1" 500 -
2019-04-30 08:29.17 Request Failed                 code=404 reason=Not Found request_json={'query': {'bool': {'should': [{'term': {'en': {'value': 'india', 'boost': 3.0}}}, {'match': {'en.trigrams': 'india'}}], 'minimum_should_match': 1, 'boost': 1.0}}, 'highlight': {'fields': {'en': {}}}}
```

This is with local pointing to suggest url. Same 500 on the deployed version.

File: output.tf [hunk 1]
```diff
@@ -1,3 +1,7 @@
 output "suggest-endpoint" {
   value = "${aws_elasticsearch_domain.suggest.endpoint}"
```
Comment (MebinAbraham) [inline output.tf:2]: Maybe?. 🤷‍♂ Copied the output to the python app as outputted (without the https scheme) as I assumed `requests` auto resolves it but guess not.
```suggestion
  value = "https://${aws_elasticsearch_domain.suggest.endpoint}"
```

**Comment by jonnyshaw89** [comment]
@MebinAbraham Because `country` isnt a valid list?

File: output.tf (hunk 1 above)
Comment (jonnyshaw89) [reply output.tf:2]: im confused

**Comment by MebinAbraham** [comment]
Seems like no indexes got loaded for me.

File: output.tf (hunk 1 above)
Comment (MebinAbraham) [reply output.tf:2]: Copied the suggest endpoint output given by terraform as the `LOOKUP_URL` without the https part as I assumed the request library still resolved it but guess not.

**Comment by MebinAbraham** [comment]
Have torn down and reapplied which was successful but still no indexes.

**Comment by MebinAbraham** [comment]
As found out, need to mention the prerequisite of having the `jq` command.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
I've spoken to @bameyrick and the JSON source files are something we've created from some original CSVs and Google translate. For now we should commit the built code lists and remove the source ones. Longer term we will probably need to create a CSV to ElasticSearch index converter.

File: built-code-lists/ethnic-groups-arab.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
+{"index" : {}}
```
Comment (ajmaddaford) [inline built-code-lists/ethnic-groups-arab.json]: I don't think the folder should be called `built-code-lists` now we're not building them, and there are no codes. Something like `index-data` would be a clearer name.

Also, what's the distinction between these indexes and the files in the `data` folder?

**Review by MebinAbraham — APPROVED** [review]

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**